app/gui/src/project-view/components/widgets/DropdownWidget.vue

```diff
@@ -11,7 +11,10 @@ enum SortDirection {
 }
 
 const props = defineProps<{ color: string; backgroundColor: string; entries: Entry[] }>()
-const emit = defineEmits<{ clickEntry: [entry: Entry, keepOpen: boolean] }>()
+const emit = defineEmits<{
+  clickEntry: [entry: Entry, keepOpen: boolean, target: HTMLElement]
```

Comment:
I would name last arg `htmlElement` as first argument may be called a target too.